common/chunkStatusLogger.go

```diff
@@ -265,6 +267,7 @@ func NewChunkStatusLogger(jobID JobID, cpuMon CPUMonitor, logFileFolder string,
 		outputEnabled:  enableOutput,
 		unsavedEntries: make(chan *chunkWaitState, 1000000),
 		flushDone:      make(chan struct{}),
+		done:           make(chan struct{}, 1),
```

Comment:
pls add a comment

// Signalling channel through which CloseLogger() conveys end-of-logging to the main logger goroutine.